src/lsp/ext.rs

```diff
@@ -91,6 +91,22 @@ impl Notification for ProjectConfigParsingFailed {
     const METHOD: &'static str = "cairo/projectConfigParsingFailed";
 }
 
+/// Notifies that both `cairo_project.toml` and `Scarb.toml` were found for the crate.
+#[derive(Debug)]
+pub struct ProjectManifestsConflict;
+
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct ProjectManifestsConflictParams {
```

Comment:
I wonder, can't we use https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#window_showMessage instead of creating all these custom notifications?

Reply:
Will try it

Reply:
Yep, it's much better. Will migrate the rest of messages in the next PR